evm_loader/lib/src/types/mod.rs

```diff
@@ -27,6 +35,54 @@ pub struct ChDbConfig {
     pub clickhouse_password: Option<String>,
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone, Default)]
+pub struct RocksDbConfig {
+    pub rocksdb_host: String,
+    pub rocksdb_port: u16,
+}
+
```

Comment:
I suggest adding enum with two options: ChDbConfig and RocksDbConfig. And method that reads this enum from environment variable.